Essentials from the following:
commit d025f3deb6349f56a7fc49551e819cfe13f97917
Author: Paul Beckingham <paul@beckingham.net>
Date:   Wed Aug 31 17:48:45 2016 -0400

    TW-1778: Unicode strings are truncated in task description
    
    - Thanks to Andrew, bjonnh, OKOMPer, Vladimir.

--- a/src/text.cpp
+++ b/src/text.cpp
@@ -248,7 +248,7 @@ bool extractLine (
     // Premature EOL.
     if (character == '\n')
     {
-      line = text.substr (offset, line_length);
+      line = text.substr (offset, prior_cursor - offset);
       offset = cursor;
       return true;
     }
